scripts/scil_compute_ihMT_maps.py

```diff
@@ -0,0 +1,359 @@
+# -*- coding: utf-8 -*-
+"""
+Computing ihMT and non-ihMT maps
+Make a more details description !
+"""
+
+import argparse
+import os
+
+import json
+import nibabel as nib
+import numpy as np
+import scipy.ndimage
+
+from scilpy.io.utils import (add_overwrite_arg,
+                             assert_output_dirs_exist_and_empty)
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(description=__doc__,
+                                formatter_class=argparse.RawTextHelpFormatter)
+    p.add_argument('in_echoes', type=int,
+                   help='Number of echoes by contrasts')
+    p.add_argument('id_subj',
+                   help='Subject name for saving maps')
+    p.add_argument('out_dir',
+                   help='Path to output folder')
+    p.add_argument('in_segment_tissue', nargs='+',
```

Comment:
go back to a mask since it will be computed on a flow. And use `get_data_as_mask()`